vulnfeeds/cmd/combine-to-osv/main.go

```diff
@@ -42,13 +48,46 @@ func main() {
 	}
 
 	allCves := loadAllCVEs(*cvePath)
-	allParts := loadParts(*partsInputPath)
-	combinedData := combineIntoOSV(allCves, allParts, *cveListPath)
+	allParts, cveModifiedMap := loadParts(*partsInputPath, *inputBucket)
+	combinedData := combineIntoOSV(allCves, allParts, *cveListPath, cveModifiedMap)
 	writeOSVFile(combinedData, *osvOutputPath)
 }
 
+// getModifiedTime gets the modification time of a given GCS file
+func getModifiedTime(innerPath string, bucket string) (time.Time, error) {
+	// Uses the GCS JSON API to get the modified date, details: https://cloud.google.com/storage/docs/json_api/v1/objects/get
+	baseUrl := "https://storage.googleapis.com/storage/v1"
+	filePath := fmt.Sprintf("%s/b/%s/o/%s", baseUrl, bucket, url.PathEscape(innerPath))
+	var emptyTime time.Time
+	response, err := http.Get(filePath)
+
+	if err != nil {
+		return emptyTime, err
+	}
+	defer response.Body.Close()
+
+	if response.StatusCode != http.StatusOK {
+		return emptyTime, fmt.Errorf("Failed to request GCS JSON API: %s", response.Status)
+	}
+
+	type Metadata struct {
+		Updated string `json:"updated"`
+	}
+
+	var jsonData Metadata
+	if err := json.NewDecoder(response.Body).Decode(&jsonData); err != nil {
+		return emptyTime, err
+	}
+
+	modifiedTime, err := time.Parse(time.RFC3339, jsonData.Updated)
+	if err != nil {
+		return emptyTime, err
+	}
+	return modifiedTime, err
+}
+
 // loadInnerParts loads second level folder for the loadParts function
-func loadInnerParts(innerPartInputPath string, output map[cves.CVEID][]vulns.PackageInfo) {
+func loadInnerParts(innerPartInputPath string, output map[cves.CVEID][]vulns.PackageInfo, inputBucket string, cvePartsModifiedTime map[cves.CVEID]time.Time) {
```

Comment:
Can you update the doc comment with what each parameters does, now that there are quite a few parameters.